src/plugins/discord/sqliteMirror.js

```diff
@@ -497,4 +497,20 @@ export class SqliteMirror {
       .all({channel_id: channel, message_id: message})
       .map((e) => Model.refToEmoji(e.emoji));
   }
+
+  nthMessageIdFromTail(channel: Model.Snowflake, n: number): ?Model.Snowflake {
```

Comment:
nit: might be nice to have a brief comment or docstring explaining that it returns undefined when `n` is greater than the number of messages -- kind of implied by the type signature but not 100% clear

Reply:
Updated